### DISCUSSION TITLE
amrex get_line_data

### DISCUSSION THREAD

**Discussion by indra098124** [AMReX-Codes/amrex#4166] [Q&A]
I want to extract the data along a line. I understand that get_line_data needs the cell index, but I would know the coordinates of the point through which the line passes. How can I use get_line_data for this purpose. The index used by get_line_data is the index at the level 0? Does get_line_data extracts the data from the finest level?

**Comment by WeiqunZhang** [comment]
You can compute the cell index using the physical coordinates before calling get_line_data. The function is for single level. If you want to get line data on AMR grids. the way to do it might be using a Particle container and gather data onto particles.
